src/main/java/io/jenkins/plugins/endpoints/PluginEndpoint.java

```diff
@@ -55,4 +68,46 @@ public Plugin getPlugin(@PathParam("name") String name) {
     }
   }
 
+  /**
+   * <p>Get issues</p>
+   *
+   * @param name The plugin to retrieve
+   * @return Matching issues
+   */
+  @Path("/issues/open")
```

Comment:
Thinking about different issue types here, should the plugin-site or the plugin-site-api handle determining what type of issues a plugin uses for issue tracking?

i.e. should this have jira in the URL, or should the getIssue return an issue type which both jira and github issues extend?

Reply:
UI should always be dumb. API smart.

Right now the official issue system for plugins is jira. If that changes to support more options officially then the backend should get refactored.